db/batchlog_manager.cc

```diff
@@ -110,35 +112,47 @@ future<> db::batchlog_manager::do_batch_log_replay() {
     });
 }
 
+// Since replay is a "node global" operation, we should not attempt to do
+// it in parallel on each shard. It will just overlap/interfere.  To
```

Comment:
I don't agree with this moved comment.

Reply:
You don't agree with its contents or it being moved?
Changing this part is out of scope for this fix...